src/main/kotlin/com/lykke/me/subscriber/incoming/events/CashOutEvent.kt

```diff
@@ -0,0 +1,3 @@
+package com.lykke.me.subscriber.incoming.events
+
+interface CashOutEvent: MeEvent
```

Comment:
why do we need these event interfaces ?

Reply:
just marker for CashOutEvent (just in case - theoretically we can have not only proto events in the future)